src/vivarium/framework/population/manager.py

```diff
@@ -490,33 +488,38 @@ def get_population(
             for attribute in attributes_to_include
             if attribute not in simple_attributes
         ]
+        contains_column_multi_index = False
         for name in remaining_attributes:
-            pipeline = self._attribute_pipelines[name]
-            values = self._population.loc[idx, name] if pipeline.is_simple else pipeline(idx)
+            values = self._attribute_pipelines[name](idx)
+
+            # Handle column names
             if isinstance(values, pd.Series):
                 if values.name is not None and values.name != name:
                     self.logger.warning(
-                        f"The '{name} attribute pipeline returned a pd.Series with a "
+                        f"The '{name}' attribute pipeline returned a pd.Series with a "
                         f"different name '{values.name}'. For the column being added to the "
                         f"population state table, we will use '{name}'."
                     )
                 values.name = name
+            else:
```

Comment:
Alternatively, we just make every state table returned multi-index columns even though they often might not actually have any values in the second level?